lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -310,7 +354,8 @@ void OrphanedNodesManager::handleOp(const Ufe::SceneCompositeNotification::Op& o
         // point.  It may be an internal node, without data.
         auto ancestorNode = _pulledPrims.node(op.path);
         TF_VERIFY(ancestorNode);
-        recursiveSwitch(ancestorNode, op.path);
+        recursiveSwitch(ancestorNode, op.path, true);
+        recursiveSwitch(ancestorNode, op.path, false);
```

Comment:
We need to process newly orphaned nodes first because that deletes information in various places while the ones that are no longer orphans add information.  If we let the change be done in any order, we would sometimes lose information that got deleted by orphans.